caddy/setup/locale.go

```diff
@@ -0,0 +1,79 @@
+package setup
+
+import (
+	"strings"
+
+	"github.com/mholt/caddy/middleware"
+	"github.com/mholt/caddy/middleware/locale"
+	"github.com/mholt/caddy/middleware/locale/method"
+)
+
+// Locale configures a new Locale middleware instance.
+func Locale(c *Controller) (middleware.Middleware, error) {
+	locale := &locale.Locale{}
```

Comment:
In this case this works. But it is preferred not use a variable name that overrides an import.